documentation/FortranForCProgrammers.md

```diff
@@ -260,8 +260,7 @@ little like C++17's `std::visit()` on a discriminated union.
 Pointers
 --------
 Pointers are objects in Fortran, not data types.
-Pointers can point to data, arrays, and subprograms, but not to other pointers
-or to an allocatable.
+Pointers can point to data, arrays, and subprograms, but not to other pointers.
```

Comment:
What was intended by the original text is that one cannot construct a pointer in Fortran that points to the *implementation* of an allocatable object as such, as opposed to the data held within the allocatable.  One can certainly have a pointer that points to those contents, so long as the allocatable has the TARGET attribute, just as with any other data object.